File: Foundation/Basics/reverse.java
import java.util.Scanner;
public class reverse
{
    public static Scanner scn = new Scanner(System.in);

    public static void printReverse(int n)
    {
        int num = n;
        while(num != 0)
        {
            System.out.println(num % 10);
            num /= 10;
        }
    }

    public static void main(String[] args)
    {
        int n = scn.nextInt();
        printReverse(n);
    }
}

File: Foundation/Basics/reverse.cpp
#include<iostream>
using namespace std;

void printReverse(int n)
{
    int num = n;
    while(num != 0)
    {
        cout<<num % 10<<endl;
        num /= 10;
    }
}

int main()
{
    int n;
    cin>>n;
    printReverse(n);
    return 0;
}
